nsdb-core/src/main/scala/io/radicalbit/nsdb/commit_log/StandardCommitLogSerializer.scala

```diff
@@ -264,11 +264,17 @@ class StandardCommitLogSerializer extends CommitLogSerializer with TypeSupport {
           _     = readByteBuffer.get(value)
         } yield (name, typ, value)).toList
 
-        InsertEntry(db = db,
-                    namespace = namespace,
-                    metric = metric,
-                    timestamp = ts,
-                    Bit(timestamp = ts, value = 0, dimensions = createDimensions(dimensions)))
+        InsertEntry(
+          db = db,
+          namespace = namespace,
+          metric = metric,
+          timestamp = ts,
+          // TODO: PLEASE REMEMBER TO USE TAGS PROPERLY
```

Comment:
Probably we can add that tags are still in progress and experimentals

Reply:
I'm going to remove all those comments in the next PR